lib/devices/platform/concrete/_platform_info_impl.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter_base_rootstrap/devices/platform/abstract/app_platform.dart';
+import 'package:flutter_base_rootstrap/devices/platform/abstract/platform_info.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class PlatformInfoImpl extends PlatformInfo {
+  final AppPlatform appPlatform;
+
+  PlatformInfoImpl(this.appPlatform);
+
+  @override
+  get isDesktopOS =>
+      appPlatform.isMacOS || appPlatform.isLinux || appPlatform.isWindows;
```

Comment:
@amaury901130 i have a queston:
I don't know if we will use this, because sometimes there is a specific widget for a specific OS, not for specific global ones, also for the `isAppOS`

Reply:
the idea is to use this when we need to do something specific with the device that requires special permissions for example o any other thing.